syntaxes/Dml.tmlanguage

```diff
@@ -0,0 +1,1175 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>fileTypes</key>
+	<array>
+		<string>dml</string>
+	</array>
+<!-- 	<key>firstLineMatch</key>
+	<string>-[*]-( Mode:)? dml -[*]-</string> -->
+	<key>foldingStartMarker</key>
+	<string>(?x)
+		 /\*\*(?!\*)
+		|^(?![^{]*?//|[^{]*?/\*(?!.*?\*/.*?\{)).*?\{\s*($|//|/\*(?!.*?\*/.*\S))
+	</string>
+	<key>foldingStopMarker</key>
+	<string>(?&lt;!\*)\*\*/|^\s*\}</string>
+	<key>keyEquivalent</key>
+	<string>^~C</string>
+	<key>name</key>
+	<string>Dml</string>
+	<key>patterns</key>
+	<array>
+		<dict>
+			<key>include</key>
+			<string>#preprocessor-rule-enabled</string>
+		</dict>
+		<dict>
+			<key>include</key>
+			<string>#preprocessor-rule-disabled</string>
+		</dict>
+		<dict>
+			<key>include</key>
+			<string>#preprocessor-rule-other</string>
+		</dict>
+		<dict>
+			<key>include</key>
+			<string>#comments</string>
+		</dict>
+		<dict>
+			<key>match</key>
+			<string>\b(break|case|continue|default|do|else|for|goto|if|_Pragma|return|switch|while|foreach|select|where)\b</string>
+			<key>name</key>
+			<string>keyword.control.c</string>
+		</dict>
+		<dict>
+			<key>match</key>
+			<string>\b(asm|__asm__|auto|bool|_Bool|char|_Complex|double|enum|float|_Imaginary|int|long|short|signed|struct|typedef|union|unsigned|void)\b</string>

```

Comment:
These are leftover C aspects that rather obviously don't belong in DML `asm`, `__asm__`, `auto`, `_Bool`, `_Complex`, `enum`, `_Imaginary`, and `union`. However, they might still be helpful for `header %{ %}` blocks. Possibly. I don't think this blocks a merge.